app/styles/styles.css

```diff
@@ -110,3 +119,108 @@ input:focus {
   cursor: pointer;
   background: transparent;
 }
+
+.list-item-component label {
```

Comment:
If you could attach a couple of screenshots highlighting the new ui changes that would be awesome!